cranelift/codegen/src/isa/s390x/inst/regs.rs

```diff
@@ -178,6 +179,16 @@ pub fn pretty_print_reg(reg: Reg, allocs: &mut AllocationConsumer<'_>) -> String
     show_reg(reg)
 }
 
+pub fn pretty_print_regpair(pair: RegPair, allocs: &mut AllocationConsumer<'_>) -> String {
+    let hi = allocs.next(pair.hi);
+    let lo = allocs.next(pair.lo);
+    if hi.is_real() {
+        show_reg(hi)
```

Comment:
Not sure if there can be a phase where only one of `hi` and `lo` are real regs?   Also, it would be good to assert in the real reg case that we have a valid even/odd pair.

Reply:
I changed this to require that both `hi` and `lo` are real registers, and to assert that they are valid.